src/components/layout.module.css

```diff
@@ -8,6 +8,7 @@
 }
 
 .header {
+  composes: container from "gatsby-theme-landing-page/src/components/base.module.css";
```

Comment:
Just testing to see if this works with shadowing; might pull global base styles back out to the starter and keep these more decoupled